lib/manageiq/messaging/client.rb

```diff
@@ -2,6 +2,7 @@ module ManageIQ
   module Messaging
     class Client
       require 'manageiq/messaging/stomp/client'
+      require 'manageiq/messaging/kafka/client'
```

Comment:
Ideally we should only require what is used to keep memory down... Can probably do this with Kernel#autoload